model/schema/phenotypeAndDiseaseAnnotation.yaml

```diff
@@ -554,51 +552,51 @@ slots:
   has_condition:
     is_a: related_to
     symmetric: false
-    domain: Entity
+    domain: Association
```

Comment:
Not sure what the rationale for this change is (and other changes in this file), but these slots are not used anywhere.  I think these are a hangover from before we decided to use Vocabulary and VocabularyTerm classes to deal with the controlled vocabularies and should be removed altogether - unless I am missing something.

Reply:
Yep, good catch. I've just pushed a commit to delete those predicate definitions altogether